cmd/operator-sdk/build/cmd.go

```diff
@@ -225,7 +224,7 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 			case projutil.OperatorTypeHelm:
 				return fmt.Errorf("test scaffolding for Helm Operators is not implemented")
 			default:
-				return fmt.Errorf("unknown operator type '%v'", t)
+				return projutil.ErrUnknownOperatorType{}
```

Comment:
Do we want to specify the operator type (`t`) here (and in other similar locations) still or ignore it now?

Reply:
The type is inferred from project files, and the only other type possible is “unknown”, so I have `ErrUnknownOperatorType` set up to return “unknown operator type” if `Type` is not set.